src/components/shared/MobileMenu/MenuItem.tsx

```diff
@@ -91,8 +91,11 @@ export const MenuItem = ({
           typography: 'label3',
           textDecoration: 'none',
           whiteSpace: 'nowrap',
+          fontSize: '14px',
+          fontFamily:
```

Comment:
is it necessary to give particular font family ?

Reply:
yes